meps/mep-0003.md

```diff
@@ -54,6 +54,7 @@ Content of the tip directive.
 
 ````{code}
 :label: new-syntax-example
+:caption: Proposal for inline options in directives; in this case showing a `tip` directive with a label and class.
```

Comment:
@stefanv mentioned that `Program 1` isn't actually shown anywhere this adds the caption:

<img width="1552" height="384" alt="Image" src="https://github.com/user-attachments/assets/6ed5d87b-9cab-44a0-b02d-600ed9b97fab" />